.github/ISSUE_TEMPLATE/--content-proposal-.md

```diff
@@ -1,27 +1,29 @@
 ---
-name: "\U0001F4DDContent Proposal "
-about: 'I want to contribute content to amp.dev. '
+name: "\U0001F4DD Content Proposal"
+about: 'I want to contribute content to amp.dev.'
 title: 'Content proposal '
 labels: ''
 assignees: ''
-
 ---
 
-A short description of your contribution:
+## 📝 Content Proposal
+
+**Synopsis**  
```

Comment:
I think all of these sections should use `###` instead.

Reply:
Thanks again @DerekNonGeneric - I changed a few things, lmkwyt!

Reply:
Putting those questions in their own section was a great idea! The headings idea works on most templates except for the ones with longer questions/sentences. Might need to shorten them up. Also, making those horizontal lists vertical lists was excellent too. I think this is a good opportunity to use [GitHub's interactive task lists](https://help.github.com/en/github/managing-your-work-on-github/about-task-lists). Will investigate further.